jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -3,7 +3,7 @@ package server
 package jetty
 
 import com.codahale.metrics.{InstrumentedExecutorService, MetricRegistry}
-import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedConnectionFactory}
+import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedHandler, InstrumentedConnectionFactory}
```

Comment:
It's not touching it.


    > compile

    [info] Updating
{file:/Users/jonoabroad/developer/scala/oss/ohttp4s/}jetty...

    [info] Resolving org.fusesource.jansi#jansi;1.4 ...

    [info] Done updating.

    [info] Compiling 1 Scala source to
/Users/jonoabroad/developer/scala/oss/ohttp4s/jetty/target/scala-2.10/classes...

    [error]
/Users/jonoabroad/developer/scala/oss/ohttp4s/jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala:156:
not found: type     InstrumentedHandler

    [error]       val h     = new InstrumentedHandler(reg, metricPrefix)

    [error]                       ^

    [error] one error found

    [error] (jetty/compile:compileIncremental) Compilation failed

    [error] Total time: 5 s, completed 24/06/2015 2:23:25 PM

On 24 June 2015 at 14:11, Ross A. Baker <notifications@github.com> wrote:

> In jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala
> <https://github.com/http4s/http4s/pull/309#discussion_r33116098>:
>
> > @@ -3,7 +3,7 @@ package server
> >  package jetty
> >
> >  import com.codahale.metrics.{InstrumentedExecutorService, MetricRegistry}
> > -import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedConnectionFactory}
> > +import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedHandler, InstrumentedConnectionFactory}
>
> Hmm. It's right here
> <https://github.com/http4s/http4s/blob/39d674a5d4962733ece4d3a42caacadda353be75/jetty/src/main/java/org/http4s/server/jetty/InstrumentedHandler.java>.
>
>
> Can you tell if javac is running at all on that one Java file? I don't
> understand why this doesn't Just Work for you, as it does for Travis and me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/http4s/http4s/pull/309/files#r33116098>.
>